server/src/index.ts

```diff
@@ -1,16 +1,87 @@
 import express, { Request, Response } from "express";
-import { GameState } from "shared";
+import { GameState, SERVER_PORT, PlayerId, Card, NUMBER_DIGIT_CARDS_IN_GAME, ALL_DIGIT_CARDS, NUMBER_OPERATION_CARDS_IN_GAME, ALL_OPERATION_CARDS, isExtensionOfExpression} from "shared";
+import http from "http";
+import {Server} from "socket.io";
+
 
 const app = express();
-// It would be nicer if it was defined in shared/constants 
-const port = 3001;
+const server = http.createServer(app);
+const io = new Server(server, {
+    cors : {
+        origin : "http://localhost:5173" 
```

Comment:
Fajnie że to dodałeś, ale też dobrze żeby to przychodziło z contstants. To nie będzie działać jak będziemy chcieli już zrobić produkcję.